src/instrumenter.js

```diff
@@ -158,12 +159,12 @@ function getVersion (moduleBaseDir) {
 }
 
 function filename (plugin) {
-  return [plugin.name, plugin.file].filter(val => val).join('/')
+  return path.join.apply(path, [plugin.name, plugin.file].filter(val => val))
```

Comment:
I would prefer to convert all platform specific separators to forward slashes instead to avoid any incompatibility with `require`. This would also allow plugins to use simple forward slashes.